### PR TITLE
Epic 7 - Sofia Pasichko

### PR DESCRIPTION
**Pull request by SofiaPasichko**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/sofiia_pasichko/Epic7/epic_7.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            matrix2[i][j] = i * j + i;
+        }
+    }
+    for (int i = 0; i < 10; ++i) {
+        for (int j = 0; j < 10; ++j) {
+            matrix2[i][j] = i * j + i;
+        }
+        for (int j = 0; j < 9; ++j) {
+            for (int k = 0; k < 9 - j; ++k) {
+                if (matrix2[k][j] < matrix2[k + 1][j]) {
```
Comment (fm2lpnu) [inline ai_12/sofiia_pasichko/Epic7/epic_7.cpp:45]: 5 nested cycles are overkill for function

File: ai_12/sofiia_pasichko/Epic7/epic_7.cpp [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    int k;
+    double** matrix = createMatrix();
+    newFunction(matrix);
+    for (int i = 0; i < 10; ++i) {
+        for (int j = 0; j < 10; ++i) {
+            cout << matrix[i][j];
+    }
+    }
+    k = calculateSuma(column, 5);
+    cout << k;
```
Comment (fm2lpnu) [inline ai_12/sofiia_pasichko/Epic7/epic_7.cpp:79]: Tasks 4-6 are not implemented

File: ai_12/sofiia_pasichko/Epic7/epic_7.cpp [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                    matrix[k][j] = matrix[k + 1][j];
+                    matrix[k + 1][j] = temp;
+                }
+            }
+        }
+    }
+    return matrix;
+} // функція сортує стовпці по зростанню за методом бульбашки
+
+int** newFunction(int** matrix2, int A) {
```
Comment (fm2lpnu) [inline ai_12/sofiia_pasichko/Epic7/epic_7.cpp:33]: + Attention to details

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**